sljit_src/sljitNativeARM_64.c

```diff
@@ -1702,7 +1704,7 @@ SLJIT_API_FUNC_ATTRIBUTE sljit_s32 sljit_emit_op_custom(struct sljit_compiler *c
 /*  Floating point operators                                             */
 /* --------------------------------------------------------------------- */
 
-static sljit_s32 emit_fop_mem(struct sljit_compiler *compiler, sljit_s32 flags, sljit_s32 reg, sljit_s32 arg, sljit_sw argw)
+static sljit_s32 emit_fop_mem(struct sljit_compiler *compiler, sljit_ins flags, sljit_s32 reg, sljit_s32 arg, sljit_sw argw)
```

Comment:
Is flags directly used in instruction creation? If not this change does not makes sense.

Reply:
INT_SIZE or WORD_SIZE is used as the parameter call point, for example:
https://github.com/zherczeg/sljit/blob/7f28861274052a5d3975835d593fa1e56ecb7e6e/sljit_src/sljitNativeARM_64.c#L1765
https://github.com/zherczeg/sljit/blob/7f28861274052a5d3975835d593fa1e56ecb7e6e/sljit_src/sljitNativeARM_64.c#L1800
Therefore, the 'unsigned' type needs to be declared.

Reply:
my suggestion was incorrect, this was addressed already in 335ee51f, by adding a cast directly in the macro that did the originally affected shift.